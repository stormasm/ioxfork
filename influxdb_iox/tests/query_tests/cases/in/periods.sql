-- Basic query tests for measurement names that have periods in their names
-- IOX_SETUP: PeriodsInNames

-- query data
SELECT * from "measurement.one";



-- projection
SELECT "tag.one" from "measurement.one";

-- predicate
SELECT "tag.one" from "measurement.one" where "field.two" is TRUE;
