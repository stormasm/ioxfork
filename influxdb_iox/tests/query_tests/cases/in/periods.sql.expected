-- Test Setup: PeriodsInNames
-- SQL: SELECT * from "measurement.one";
+-----------+-----------+---------+---------+--------------------------------+
| field.one | field.two | tag.one | tag.two | time                           |
+-----------+-----------+---------+---------+--------------------------------+
| 1.0       | true      | value   | other   | 2021-01-01T00:00:01.000000001Z |
| 1.0       | false     | value2  | other2  | 2021-01-01T00:00:01.000000002Z |
+-----------+-----------+---------+---------+--------------------------------+
-- SQL: SELECT "tag.one" from "measurement.one";
+---------+
| tag.one |
+---------+
| value   |
| value2  |
+---------+
-- SQL: SELECT "tag.one" from "measurement.one" where "field.two" is TRUE;
+---------+
| tag.one |
+---------+
| value   |
+---------+